src/components/chat/MessageBox.js

```diff
@@ -14,66 +14,15 @@ class MessageBox extends Component {
       messagesEnd: null
     };
   }
-  componentDidUpdate(prevProps){
-    if (this.shouldScroll(prevProps)){
-      this.scrollToBottom();
-    }
-  }
-
-  shouldScroll(prevProps){
-    let hasNewMessages = this.hasNewMessages(prevProps);
-    if (hasNewMessages){
-      this.checkNewMessages(prevProps.messages);
-    }
-    return hasNewMessages;
-  }
-
-  checkNewMessages(prevMessages){
-    let messageIds = prevMessages.map( (message) => {
-      return message.key;
-    });
-    messageIds = messageIds.concat(this.state.notifiedIds);
-
-    let newMessages = this.props.messages.filter((message) => {
-      return messageIds.indexOf(message.key) === -1;
-    });
-
-    // better logic needed, but this will play a notification if you've been mentioned.
-    let newMessageIds = [];
-    let iWasMentioned = false;
-    let newMessageFromNotMe = false;
-    newMessages.forEach( (message) => {
-      newMessageIds.push(message.key);
-      let content = message.msg.toLowerCase();
-      let username = this.props.username.toLowerCase();
-      if (content.indexOf('@' + username) > -1){
-        iWasMentioned = true;
-      }
-      if (message.from !== this.props.username){
-        newMessageFromNotMe = true
-      }
-    });
-
-    if (newMessageFromNotMe){
-      playNotification();
-    }
-    if (iWasMentioned){
-      // TODO: Display dialog box with this data:
-      // {'title': message.from, 'message': message.msg}
-    }
-
-    let notifiedIds = this.state.notifiedIds;
-    this.setState({
-      notifiedIds: notifiedIds.concat(newMessageIds)
-    });
-  }
 
-  hasNewMessages(prevProps){
-    return prevProps.messages.length !== this.props.messages.length;
+  scrollToBottom = () => {
+    playNotification();
+    this.messagesEnd && this.messagesEnd.scrollIntoView({behavior: "smooth"});
```

Comment:
@Spetastian if you remove `{behavior: "smooth"}` it automatically starts at the bottom on page load.

Reply:
@Spetastian Want to make this tweak, then I'll merge?